README.md

```diff
@@ -743,7 +741,7 @@ lookup_table(name="ip_lookup" file="/etc/rsyslog.d/tables/ip_lookup.json" reload
 ```
 
 NOTE: This does not create the actual `lookup()` call in the Rsyslog configuration file(s). Currently that is only supported via
-the `rsyslog::server::custom_config` and `rsyslog::client::custom_config` resources as it requires setting rsyslog variables (I.E. - `set $.iplook = lookup('ip_lookup', $hostname)`).
+the `rsyslog::config::custom_config` hash as it requires setting rsyslog variables (I.E. - `set $.iplook = lookup('ip_lookup', $hostname)`).
```

Comment:
Would be good when someone can take a second look here because i am not sure if that is the right replacement for the old `rsyslog::server::custom_config and rsyslog::client::custom_config resources` reference here.

Reply:
Looks right to me.